cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -795,6 +795,29 @@ __device__ inline numeric::decimal64 const column_device_view::element<numeric::
 }
 
 namespace detail {
+
+/**
+ * @brief Convenience function to get offset word from a bitmask
+ *
+ * @see copy_offset_bitmask
+ * @see offset_bitmask_and
+ */
+__device__ inline bitmask_type get_mask_offset_word(bitmask_type const* __restrict__ source,
```

Comment:
May be this is better place `include/cudf/detail/null_mask.hpp`

Reply:
It would be preferable to have `__device__` functions to reside in a cuh file rather than an hpp. `null_mask.hpp` file is entirely composed of host callable functions. While I agree that this particular function concerns null_masks it would also make sense to move it to `column_device_view.cuh` because I think @devavret plans to make `get_mask_offset_word()` a member of `column_device_view` class similar to `column_device_view::get_mask_word()`

Reply:
@kaatish , I pinged @jrhemstad about this and he gave a go ahead. `get_mask_offset_word` should be added to `column_device_view` while contributing towards #6893